app/controllers/costlog_controller.rb

```diff
@@ -268,4 +268,10 @@ def update_cost_entry_from_params
 
     @cost_entry.attributes = permitted_params.cost_entry
   end
+
+private
```

Comment:
Indent access modifiers like `private`.<br>Keep a blank line before and after `private`.